model/gen-ai/spans.yaml

```
@@ -188,6 +195,32 @@ groups:
         requirement_level:
           conditionally_required: If provided by the application.
 
+  - id: span.gen_ai.invoke_agent.client
+    type: span
+    stability: experimental
+    span_kind: client
+    brief: >
+      Describes GenAI agent invocation and is usually applicable when working
+      with remote agent services.
+    note: |
+      The `gen_ai.operation.name` SHOULD be `invoke_agent`.
+      The **span name** SHOULD be `invoke_agent {gen_ai.agent.name}` if `gen_ai.agent.name` is readily available.
+      When `gen_ai.agent.name` is not available, it SHOULD be `invoke_agent`.
+      Semantic conventions for individual GenAI systems and frameworks MAY specify different span name format.
+    extends: trace.gen_ai.client.common_attributes
+    attributes:
+      - ref: gen_ai.system
+        requirement_level: required
+      - ref: gen_ai.agent.id
+        requirement_level:
+          conditionally_required: if applicable.
+      - ref: gen_ai.agent.name
+        requirement_level:
+          conditionally_required: when available
+      - ref: gen_ai.agent.description
+        requirement_level:
+          conditionally_required: when available
```

```suggestion
          conditionally_required: when available
      - ref: gen_ai.agent.conversation_id
        requirement_level:
          conditionally_required: when available 
```

AWS bedrock `invoke_agent` API has the concept of reusing a `sessionId` param in order to track a conversation (https://docs.aws.amazon.com/bedrock/latest/APIReference/API_agent-runtime_InvokeAgent.html). Does it make sense to add it?

---

Good comment, @xrmx I created an issue here https://github.com/open-telemetry/semantic-conventions/issues/2024 for follow up.

I was hoping we can merge this PR first, as it has been for month, and I also have some other follow up PRs as well, hope it is ok, thanks!

@lmolkova ^^